webapp/src/Entity/PrefixedExternalIdInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php declare(strict_types=1);
+
+namespace App\Entity;
+
+/**
+ * Entities implementing this interface will have their external ID prefixed with 'dj-'.
+ */
+interface PrefixedExternalIdInterface
```

Comment:
```suggestion
interface PrefixedExternalIdWithDJInterface
```

It makes that we can remove the comment but I'm not sure if this is really better.

Reply:
I don't really like it.